scripts/dev

```diff
@@ -172,13 +190,13 @@ multitask test: ["test:go", "test:js"]
 
 namespace :test do
   desc "Runs Go tests"
-  task :go do
+  task :go => "db:clean" do
```

Comment:
This change (and the following one) mean that you can run `scripts/dev test:go` or e.g. `scripts/dev test:go:only ./pkg/graph/` over and over without the lifecycle ID machinery breaking on you.